.github/workflows/integration-tests.yml

```diff
@@ -10,17 +10,18 @@ jobs:
     check-token:
         name: Check secrets access
         runs-on: ubuntu-latest
+        environment: "test-trigger-is"
         outputs:
             has_token: ${{ steps.set-token-status.outputs.has_token }}
         steps:
-        - name: Check if GITHUB_TOKEN is set
+        - name: Check if DECO_WORKFLOW_TRIGGER_APP_ID is set
```

Comment:
GITHUB_TOKEN is sometimes set for external contributors but with a smaller set of permissions. It is not reliable.